examples/Backtesting/transfer_learning.py

```diff
@@ -0,0 +1,120 @@
+### Example for a simulation loop using the transfer learning capabilities
```

Comment:
after some more thought here a proposal for the overall content of the transfer learnign examples:

1) The actual demo of how it would be used in practice - not backtesting
- generate searchspae with a taskparaemter that has `values = ['old_campaign1','old_campaign2','current_campaign']` and `active_values = ['current campaign']` (or better examples closer to the real world)
- add the existsing data with `add_measurements`
- now performa a small recommendation loop and demosntrate that we get recommendations for `current_campaign`


2) A backteting example
- basically whats in here
- but also demonstrating the improvement the result have when compared to simualting without transfer learning (ie 2 additional curves)

Reply:
The current file is what you propose as 2nd file here. I will not resolve this comment until I have implemented a variant of the first demo you mention here.

Reply:
? that should be straighforward

- Tl parameter should not be excluded from searchspace but instead have `active_values='name_of_the_particular_function'`
- This will result in 3 independen campaign objects
- These can be simulated with `simulate_scenarios`
- If you use a dataframe as lookup then they can share one single dataframe as lookup

Id say its not really acceptable to code your own look as this example is about our backtest utilities